assets/css/main.css

```diff
@@ -1,11 +1,10 @@
 /* =============================================================================
 HTML5 CSS Reset Minified - Eric Meyer
```

Comment:
Yeah, this is a modified version of Eric Meyer's reset that I have played with, so I don't want to use a CDN.